inspector/src/components/actionTabs/lines/textureLineComponent.tsx

```diff
@@ -49,28 +49,28 @@ export class TextureLineComponent extends React.Component<ITextureLineComponentP
         this.updatePreview();
     }
 
-    updatePreview() {
+   async updatePreview() {

```

Comment:
Hope using async/await is okay here...I thought it was cleaner than Promises

Reply:
it is no problem at all (we do not support IE for the inspector)